aidial_sdk/application.py

```diff
@@ -188,18 +212,26 @@ async def _chat_completion(
     async def _healthcheck() -> JSONResponse:
         return JSONResponse(content={"status": "ok"})
 
-    def _get_deployment(self, deployment_id: str) -> ChatCompletion:
+    def _get_chat_completion(self, deployment_id: str) -> ChatCompletion:
         impl = self.chat_completion_impls.get(deployment_id, None)
-
         if not impl:
-            raise DIALException(
-                status_code=404,
-                code="deployment_not_found",
-                message="The API deployment for this resource does not exist.",
-            )
+            raise self._get_missing_deployment_error()
+        return impl
 
+    def _get_embeddings(self, deployment_id: str) -> Embeddings:
+        impl = self.embeddings_impls.get(deployment_id, None)
+        if not impl:
+            raise self._get_missing_deployment_error()
         return impl
 
+    @staticmethod
+    def _get_missing_deployment_error() -> DIALException:
```

Comment:
let's move this methods inside of distinct private module to reduce amount of code inside of `DialApp` class

Reply:
fixed